docs/reference/index.md

```diff
@@ -0,0 +1,24 @@
+<!--[metadata]>
++++
+title = "Notary Configuration"
+description = "Configuring the notary client, server and signer."
+keywords = ["docker, notary, notary-client, notary-server, notary server, notary-signer, notary signer"]
+[menu.main]
+identifier="mn_notary_config"
+parent="mn_notary"
+weight=90
++++
+<![end-metadata]-->
+
+# Notary configuration files
+
+This document is for power users of the [notary client](../advanced_usage.md),
+or for those who are [running their own service](../running_a_service.md) who
+want to facilitate CLI interaction or specify custom options.
+
+You can configure the following aspects of Notary:
+
+* [Server Configuration File](server-config.md)
+* [Notary Client Configuration File](client-config.md)
```

Comment:
Should the client section precede the server and signer sections so that they are grouped together (might make more sense since the next section is common only to the server and signer).  Also, should it be `Notary Server Configuration FIle` to match the client and signer?